css/main.css

```diff
@@ -0,0 +1,641 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body.active {
+  overflow-y: hidden;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.menu-button {
+  cursor: pointer;
+}
+
+.menu-desktop {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 24px;
+  height: 3px;
+  margin: 4px auto;
+  background-color: green;
+  -webkit-transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu-button.active .bar:nth-child(1) {
+  transform: translateY(7px) rotate(45deg);
+  background-color: #fff;
+}
+
+.menu-button.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.menu-button.active .bar:nth-child(3) {
+  transform: translateY(-7px) rotate(-45deg);
+  background-color: #fff;
+}
+
+.menu-desktop.active {
+  display: flex;
+  flex-direction: column;
+}
+
+.pop {
+  position: absolute;
+  flex-direction: column;
+  text-align: start;
+  width: 100%;
+  height: 100vh;
+  left: -100%;
+  top: 0;
+  background-color: #3c3a39;
+  transition: 0.3s;
+}
```

Comment:
- [ ] Please I suggest you follow the [Figma Design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1-1471) as it is one of the project requirements

_**YOUR DESIGN**_

 

- [ ] When you click on the hamburger menu, this is the design displayed

![Screenshot (70)](https://user-images.githubusercontent.com/78104544/225483926-31c6277c-6902-4406-99e6-f024e11aa739.png)

_**FIGMA PROJECT DESIGN**_

- [ ] When the hamburger menu is clicked the design that should be displayed

![Screenshot (68)](https://user-images.githubusercontent.com/78104544/225482310-f6a65988-fcba-49ff-9561-b6e8c4b5d0aa.png)